src/openforms/js/components/admin/form_design/variables/registration/RegistrationsSummaryList.js

```diff
@@ -35,7 +36,12 @@ const RegistrationSummary = ({
   const [modalOpen, setModalOpen] = useState(false);
 
   return (
-    <>
+    <ErrorBoundary
```

Comment:
I think moving it inside the modal is best, since the errors trigger during opening the modal, no?

```jsx
<ErrorBoundary>
  {variableConfigurationEditor}
  <SubmitRow>...</SubmitRow>
</ErrorBoundary>
```
etc.

Reply:
I added it here as we may also have a crash when rendering the summary (for Objects API this the `path > to > target` thing)